### PR TITLE
Create item

### PR DESCRIPTION
**Pull request by kudukacar**
This pull request gives the user the ability to add an item to the inventory.  The user adds an item through a form, and is then redirected to the homepage to see the added item in the inventory.  There is no validation of information entered or error messages.  If a user enters an item incorrectly, the inventory will simply not reflect the item.  This PR addresses the last user story in the frontend Gilded Rose project.  See screencast below.

https://user-images.githubusercontent.com/41526816/115775749-3f1e6f80-a381-11eb-890b-2c441994927e.mov

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -11,7 +13,10 @@ const App = () => {
         <Switch>
           <Route 
             exact path="/" 
-            component={() => <FetchItems component={ItemsIndex} />}/>
+            component={() => <FetchItems Component={ItemsIndex} />}/>
+          <Route 
+            exact path="/additem" 
+            component={() => <CreateItem Component={Form} />}/>
```
Comment (damonkelley) [inline frontend/src/App.tsx:19]: **praise**: ❤️

**Review by damonkelley — APPROVED** [review]
Looks great!!

File: frontend/src/form.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          <input
+              title="quality"
+              type="number"
+              name="quality"
+              value={createItemValues.quality === null ? "" : createItemValues.quality}
+              onChange={(e) => setCreateItemValues({...createItemValues, quality: e.target.value})}
+              required
+            />
+        </div>
+        <input aria-label="submit" type="submit" name="submit" value="SUBMIT"/>
```
Comment (hyoyou) [inline frontend/src/form.tsx:59]: praise: nice work adding an aria-label for a11y!

File: frontend/src/form.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+              type="number"
+              name="sellin"
+              value={createItemValues.sellIn === null ? "" : createItemValues.sellIn}
+              onChange={(e)=> setCreateItemValues({...createItemValues, sellIn: e.target.value})}
+              required
+            />
+        </div>
+        <div>
+          <label>quality:
+          </label>
```
Comment (hyoyou) [inline frontend/src/form.tsx]: nitpick: super nitpicky, but it's a little odd-looking to me that the closing tag for `label` is on a new line. Is that how the linter is set up?

**Review by hyoyou — APPROVED** [review]
🌟

File: frontend/src/form.tsx (hunk 3 above)
Comment (kudukacar) [reply frontend/src/form.tsx]: It shouldn't be on a different line.  I'll adjust.

**Pull request merged**